### PR TITLE
update co to v2.0.2

### PR DESCRIPTION
**Pull request by idealvin**
**Describe the pull request**

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  <linux, windows, osx>, <Yes>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  <Yes / I am still working on this PR>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/co/install-dll.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -110,6 +110,7 @@ endif()
+ 
+ install(
+     TARGETS co
+-    LIBRARY DESTINATION lib   # shared lib installed to   ${CMAKE_INSTALL_PREFIX}/lib
+-    ARCHIVE DESTINATION lib   # static lib installed to   ${CMAKE_INSTALL_PREFIX}/lib
++    LIBRARY DESTINATION lib
++    ARCHIVE DESTINATION lib
```
Comment (dg0yt) [inline ports/co/install-dll.patch:12]: To minimize the patch, you shouldn't remove the comments from these lines.

Comment (idealvin) [reply ports/co/install-dll.patch]: Fixed..

**Comment by strega-nil-ms** [comment]
Assigning to myself for further review.

File: ports/co/portfile.cmake [hunk 2]
```diff
@@ -21,7 +22,10 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    OPTIONS ${FEATURE_OPTIONS}
+    OPTIONS
+        ${FEATURE_OPTIONS}
+        -DSTATIC_VS_CRT=${STATIC_CRT}
+    DISABLE_PARALLEL_CONFIGURE
```
Comment (ras0219-msft) [inline ports/co/portfile.cmake:28]: It's unfortunate that we have to disable parallel configure here -- Windows builds are especially improved by being able to run both configure steps at the same time.

**Review by ras0219-msft — APPROVED** [review]
Thanks for the update!

File: ports/co/portfile.cmake (hunk 2 above)
Comment (BillyONeal) [reply ports/co/portfile.cmake:28]: That's true, but I don't think we should block a version update over it.

**Comment by BillyONeal** [comment]
Thanks for the update!

File: ports/co/portfile.cmake (hunk 2 above)
Comment (idealvin) [reply ports/co/portfile.cmake:28]: > It's unfortunate that we have to disable parallel configure here -- Windows builds are especially improved by being able to run both configure steps at the same time.

I'll remove it next time if it works.

**Comment by idealvin** [comment]
Thanks for the merge!

**Pull request merged**